### ISSUE TITLE
Is key mapping to locate elements configurable

### ISSUE THREAD

**Issue by cnachiappan** [appium/appium-for-mac#70]
To locate an element using appium for mac, place the cursor and press fn key.
Is this key mapping configurable.

Running into issues while trying to locate elements on mac VM

**Comment by reder9** [comment]
Yes, this so much. I'm running into the same issue myself and its so frustrating because you cant send the function key thru VMWare.
